tests/yast2_cmd/yast_storage.pm

```diff
@@ -0,0 +1,84 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: yast storage test
+# List disks and list partitions
+# Maintainer: Michael Grifalconi <mgrifalconi@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+    zypper_call "in yast2-storage";
+    #    assert_script_run 'yast disk list disks';
+    #    assert_script_run 'yast disk list partitions';
+    #
+    my $lsblk_output      = script_output("lsblk");
+    my $list_disks_output = script_output("yast disk list disks");
+    my $list_part_output  = script_output("yast disk list partitions");
+
+    # Checks that all disks listed by 'lsblk' are part of 'yast disk list disks'
+    my $current_test_device;
+    my $found;
+    my @tested_devices;
+    for my $lsblk_line (split /^/, $lsblk_output) {
+        if ($lsblk_line =~ /^([a-z]d[a-z])(.*)disk/) {
+            $current_test_device = $1;
+            $found               = 0;
+            for my $list_disks_line (split /^/, $list_disks_output) {
+                if ($list_disks_line =~ /^(.*)$current_test_device/) {
+                    $found++;
+                    push @tested_devices, $current_test_device;
+                }
+            }
+            die "Could not find disk match between lsblk and yast disk list disks" unless ($found);
+        }
+    }
+    die "Could not find a valid block device to test" unless (@tested_devices);
+
+    print "Tested devices:\n";
```

Comment:
This print is for who ? it will be not visible, the perl code is not executed directly on VM.

Reply:
Valid point, here is some context:
Comes from https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10629 I did, based on https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7280/ 
In the initial PR (second link) there was no mention about issues with print so I did not think much about it. It was useful for local development but I missed to realize that was not being printed on the real test execution.

Thanks for pointing it out, I will remove all `print` statements